### PR TITLE
[AFD] Adds the brainalyzer! New psychiatrist advanced machine!

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Adds the brainalyzer, a machine found over the psych office that is capable of analyzing the brain of patients.. maybe one day it will replace bald therapists...

Sounds by BlackDog
Got sprite from https://github.com/ParadiseSS13/Paradise/pull/19969
## Why It's Good For The Game

More accurate diagnosis on the mental station of the crew!
## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/77684085/e69770b1-1019-46f3-978b-4f7f36c29e25

## Testing

- Its a research station, like NT would do.. lets test it live!
## Changelog
:cl: Henri215, BlackDog
add: The brainalyzer machine has been added to all psychiatrist offices, place your patients there to receive an accurate diagnosis on their mental state!
/:cl:

### CODE REVIEW

**Review by Contrabang — CHANGES_REQUESTED** [review]
oh dear lord, can you make this not a subtype of the bodyscanner

File: code/game/machinery/adv_med.dm [hunk 1]
```diff
@@ -146,9 +146,9 @@
 		return TRUE
 
 	if(H == user)
-		visible_message("[user] climbs into [src].")
+		visible_message("[user] climbs into the body scanner.")
```
Comment (Henri215) [inline code/game/machinery/adv_med.dm:149]:
```suggestion
		visible_message("[user] climbs into [src].")
```

File: code/game/machinery/adv_med.dm [hunk 2]
```diff
@@ -146,9 +146,9 @@
...
 	else
-		visible_message("[user] puts [H] into the body scanner.")
+		visible_message("[user] puts [H] into [src].")
```
Comment (Henri215) [inline code/game/machinery/adv_med.dm:151]:
```suggestion
		visible_message("[user] puts [H] into the body scanner.")
```

**Review by Contrabang — APPROVED** [review]
FISI

**Pull request merged**